web/thesauruses/ruby/data_types.json

```diff
@@ -0,0 +1,86 @@
+{
+  "meta": {
+    "language": "ruby",
+    "language_version": "3.0",
+    "language_name": "Ruby"
+  },
+  "categories": {
+    "Numerical": [
+      "boolean",
+      "integer_8_bit",
+      "integer_16_bit",
+      "integer_32_bit",
+      "integer_64_bit",
+      "integer_as_object",
+      "float_16_bit",
+      "float_32_bit",
+      "float_64_bit",
+      "float_as_object",
+      "char"
+    ],
+    "Strings": [
+      "string_as_object",
+      "string_as_array"
+    ],
+    "Complex Numbers": [
+      "complex_as_object",
+      "real_number_part",
+      "imaginary_number_part"
+    ]
+  },
+  "data_types": {
+    "boolean": {
+      "name": "",
```

Comment:
All the `name` fields can actually be removed from this file. They're not loaded into the site.

Reply:
Thank you! I will push a new commit to fix that!